### PR TITLE
Add validator for invalid/empty url

### PR DESCRIPTION
**Pull request by shawnk0723**

### CODE REVIEW

File: src/common/ArticleForm.tsx [hunk 1]
```diff
@@ -4,29 +4,36 @@ import globalReadings from '../features/globalReadings';
...
 }
 
 class ArticleForm extends Component<IProps, IState> {
     constructor(props: IProps) {
         super(props);
         this.state = {
             url: '',
-            tags: ''
+            tags: '',
+            errors: ''
```
Comment (aTmb405) [inline src/common/ArticleForm.tsx:25]: See other comment - errors are handled in the redux store.

**Review by aTmb405 — CHANGES_REQUESTED** [review]

**Comment by shawnk0723** [comment]
Refactored validator for invalid/empty url to use addAlert() action to dispatch an error alert message. Attached the screenshots as well. Please let me know if there's any other issues with this update. 

validator2

validator1

File: src/common/ArticleForm.tsx [hunk 2]
```diff
@@ -40,8 +47,18 @@ class ArticleForm extends Component<IProps, IState> {
         }
     };
     
+    
     handleNewUrl = (e: React.FormEvent<HTMLFormElement>): void => {
+      
+   
         e.preventDefault();
+        if (validator.isURL(this.state.url)) { 
+            this.setState({errors: ''})
```
[the remark this answers is not in the document]
Comment (shawnk0723) [reply src/common/ArticleForm.tsx]: Hey Austin, fixed the code per your request. Please review and let me know if there's any other issues with it.

**Review by aTmb405 — APPROVED** [review]
Looks great! Thanks for the changes

**Pull request merged**